data/calculated.py

```diff
@@ -121,13 +121,13 @@ def __init__(self, parent, expression, dims, attrs={}, db_url=""):
         for _ in self._parser.lexer.lexer:
             pass
 
-    def __getitem__(self, key):
+    def __getitem__(self, key: str) -> xr.DataArray:
         # This is where the magic happens.
 
         data_array = self._parser.parse(
             self._expression, self._parent, key, self._dims)
 
-        return xr.DataArray(data_array)
+        return xr.DataArray(data_array, attrs=self.attrs)
```

Comment:
Fixed two year old bug where DataArray generated from calculation layer didn't have the original variable's attributes LOL